src/game/frontend/submenus/World/weather.hpp

```diff
@@ -0,0 +1,34 @@
+#include <game/rdr/Natives.hpp>
+inline const char* WeatherTypes[]{
+
+    "BLIZZARD",
+    "CLOUDS",
+    "DRIZZLE",
+    "FOG",
+    "GROUNDBLIZZARD",
+    "HAIL",
+    "HIGHPRESSURE",
+    "HURRICANE",
+    "MISTY",
+    "OVERCAST",
+    "OVERCASTDARK",
+    "RAIN",
+    "SANDSTORM",
+    "SHOWER",
+    "SLEET",
+    "SNOW",
+    "SNOWLIGHT",
+    "SUNNY",
+    "THUNDER",
+    "THUNDERSTORM",
+    "WHITEOUT",
+};
+
+void RDR2_ChangeWeather(const char* weather)
```

Comment:
I would remove the rdr2 at the beginning of the function name

Reply:
ok